src/roktManager.ts

```diff
@@ -126,6 +128,8 @@ export default class RoktManager {
         // Launcher options are set here for the kit to pick up and pass through
         // to the Rokt Launcher.
         this.launcherOptions = { sandbox, ...options?.launcherOptions };
+
+        this.domain = options?.domain;
```

Comment:
Nit: Not sure if we should attach `domain` if we don't have to.

```suggestion
        if (options?.domain) {
            this.domain = options?.domain;
        }    
```

Reply:
updating slightly, but will include